holoviews/plotting/renderer.py

```diff
@@ -275,7 +286,7 @@ def _apply_post_render_hooks(self, data, obj, fmt):
         return data
 
 
-    def html(self, obj, fmt=None, css=None, **kwargs):
+    def html(self, obj, fmt=None, css=None, resources='cdn', **kwargs):
```

Comment:
Any reason 'cdn' is lowercase here? Isn't it an acronym that is normally capitalized?

Reply:
Fixed.